### PR TITLE
Implement a UI component for editing profile photo

### PR DESCRIPTION
**Pull request by hayderjebur**
### What this PR does (required):
- list PR functionality here
- Render the UI for uploading profile photos.
- Add temporary button for display the Upload component conditinaly.
### Screenshots / Videos (front-end only):
- Post at least one screensho
Screen Shot 2021-10-13 at 4 38 11 PM
t to show the feature in action. **Only required in front-end work**
- 

### Any information needed to test this feature (required):
- Post what steps to take to be able to test this feature (eg. "sign up as a new user and upload a profile picture from the navbar")
- 

### Any issues with the current functionality (optional):
- List any problems where you were not able to complete all tasks on ticket
- List any problems this PR may cause for the project or other active PRs
- Post a screenshot/video of the problem, along with detailed console outputs where applicable

### CODE REVIEW

**Comment by DestinTech** [comment]
Best practice is to merge in to develop, not main. You can change this on this page, there's a dropdown that says main.


 I just submitted "feature_profile_settings_skeleton_#25", It is the skeleton that you will need to implement this under. Maybe wait until that one is merged in to develop and pull it from there before pushing this. This looks great though!
 
 
Screen Shot 2021-10-14 at 12 38 23 AM
Screen Shot 2021-10-14 at 12 39 34 AM

File: client/src/components/UploadPhoto/UploadPhoto.tsx
Comment (ezeilo-su) [inline client/src/components/UploadPhoto/UploadPhoto.tsx:29]: Remove this comment here :smiley:

File: client/src/components/UploadPhoto/useStyles.ts
Comment (ezeilo-su) [inline client/src/components/UploadPhoto/useStyles.ts:8]: Remove unused line of code

**Review by ezeilo-su — COMMENTED** [review]
This is a good beginning :+1: 
See a few comments left.

File: client/src/components/UploadPhoto/UploadPhoto.tsx
Comment (ugur-akin) [inline client/src/components/UploadPhoto/UploadPhoto.tsx:15]: You can probably add these styles directly to the `Box` components through its props.

File: client/src/components/UploadPhoto/UploadPhoto.tsx
Comment (ugur-akin) [inline client/src/components/UploadPhoto/UploadPhoto.tsx:42]: This looks really nice 👍🏼 I would only suggest editing the `id/htmlFor` fields to fit our case better. It might also be a good idea to abstract away this into a reusable component! (this is optional)

File: client/src/components/UploadPhoto/UploadPhoto.tsx
Comment (ugur-akin) [inline client/src/components/UploadPhoto/UploadPhoto.tsx:44]: I generally like to rename the icons to reflect their functional purpose rather than their original name (e.g. if I import `AccessTimeRoundedIcon` as an icon to add to my `TimePicker`, I might rename it to `TimePickerIcon`) 🙌🏼

File: client/src/components/UploadPhoto/UploadPhoto.tsx
Comment (ugur-akin) [inline client/src/components/UploadPhoto/UploadPhoto.tsx:46]: It would be healthier to use a `Button` to implement this section of the UI.

File: client/src/pages/Dashboard/Dashboard.tsx
Comment (ugur-akin) [inline client/src/pages/Dashboard/Dashboard.tsx:46]: Let's either remove this testing code from the PR or leave a `TODO` comment to remind about removing this.

File: client/src/components/UploadPhoto/UploadPhoto.tsx
Comment (ugur-akin) [reply client/src/components/UploadPhoto/UploadPhoto.tsx:29]: I think this is a fine comment to leave in, I guess would make it a `NOTE` instead of a `TODO`. Maybe move it outside of the JSX though unsure if this improves anything 🤔

**Review by ugur-akin — CHANGES_REQUESTED** [review]
Good work so far! Please address any comments @ezeilo-su or I left in and let's check back to merge this soon!

A couple of additional notes about the PR:
- Let's set a title that's self-explanatory and in plain English (e.g. `Implement a UI component for editing profile photo`),
- Keep the formatting of the template but remove any unused sections, items from the PR body.

This [article](https://hugooodias.medium.com/the-anatomy-of-a-perfect-pull-request-567382bb6067) goes more in depth about how best to format your PRs, have a look if you're interested!

**Comment by hayderjebur** [comment]
> Best practice is to merge in to develop, not main. You can change this on this page, there's a dropdown that says main.
> 
> I just submitted "feature_profile_settings_skeleton_#25", It is the skeleton that you will need to implement this under. Maybe wait until that one is merged in to develop and pull it from there before pushing this again. This looks great though!
> 
> Screen Shot 2021-10-14 at 12 38 23 AM
> 
> Screen Shot 2021-10-14 at 12 39 34 AM

File: client/src/components/UploadFileButton/UploadFileButton.tsx
Comment (ezeilo-su) [inline client/src/components/UploadFileButton/UploadFileButton.tsx:17]: Nice one :+1:

File: client/src/components/UploadFileButton/UploadFileButton.tsx
Comment (rajivtitus) [inline client/src/components/UploadFileButton/UploadFileButton.tsx:5]: 👍🏽

**Pull request merged**
